payload-files/sveltekit-2/_api_test-param-error_[param]--transaction.json

```diff
@@ -4,17 +4,17 @@
     "event_id": "[[ID1]]",
     "sdk": {
       "name": "sentry.javascript.sveltekit",
-      "version": "7.110.1"
+      "version": "8.0.0-beta.3"
     },
     "sent_at": "[[ISODateString]]",
     "trace": {
       "environment": "qa",
       "public_key": "[[publicKey]]",
-      "release": "016d073b9f5b47ef8e2d2cff020115bb",
+      "release": "769bfcd7ef414979b49e11a65ea1802c",
       "sample_rate": "1",
       "sampled": "true",
       "trace_id": "[[ID2]]",
-      "transaction": "GET /api/test-param-error/[param]"
+      "transaction": "GET /api/test-param-error/1337"
```

Comment:
The transaction is not parametrized in v8:  GET /api/test-param-error/1337

Non-parametrization happens only in preview mode and only with `type: transaction`. The error event with `type: event` is fine.
In dev mode it is always parametrized.